### PR TITLE
[AD-94] downloadPlacements

### PR DESCRIPTION
**Pull request by jesswinssinger**
I think I accidentally caught some of @justinhyou 's commits from the fixPull branch. Wondering if those are ever going to be merged into master or if I should roll them back. Otherwise this is ready to be merged with master!

- added downloadPlacements button, route, and action
- removed downloadPulls and downloadStudents

### CODE REVIEW

File: config/initializers/urls.rb [hunk 1]
```diff
@@ -1,23 +1,23 @@
-require './lib/google_api/drive.rb'
```
Comment (g-rocket) [inline config/initializers/urls.rb]: Why are you commenting out these lines?  Please remove these changes from your PR before merging.

**Comment by g-rocket** [comment]
It looks like you have a bunch of changes that aren't supposed to be part of this PR.  Can you remove them before merging?

File: config/initializers/urls.rb (hunk 1 above)
Comment (jesswinssinger) [reply config/initializers/urls.rb]: Hmm weird I didn’t touch those lines... might have been something I caught from Justin’s commits from the fixPulls branch? See desc! Hand discussion w admin team_2 and we plan to merge them at the same time to remedy the commits I accidentally caught w this branch.

**Comment by jesswinssinger** [comment]
@g-rocket Since it sounds like the pulls issue is still in progress, I (hopefully) just reverted the commits that don't belong in this PR. Check out the changes now and let me know if you spot anything strange. Thanks!

**Comment by g-rocket** [comment]
Looks reasonable now, as far as I can tell.

**Pull request merged**